packages/aragon-cli/src/lib/acl/typedef.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @typedef {Object} PermissionsData
+ * @property {string[]} allowedEntities
+ * @property {string} manager
+ *
+ * ACL structure sample: {
```

Comment:
@dapplion is this ACL structure sample complete?

Reply:
Until the `PermissionsData` level, I think so.